Models/Response/SearchResponse.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using System.Linq;
+using whale_spotting.Models.Database;
+using whale_spotting.Request;
+
+namespace whale_spotting.Models.Response
+{
+    public class SearchResponse
```

Comment:
We only have one search in this app, so I don't think we need a generic SearchResponse class which can be inherited from, I think we can just this file one single class which also just has a single constructor :)